lib/govuk_index/presenters/common_fields_presenter.rb

```diff
@@ -1,12 +1,14 @@
 module GovukIndex
   class CommonFieldsPresenter
+    CUSTOM_FORMAT_MAP = {
```

Comment:
So is this needed because specialist pub is currently sending
european_structural_investment_fund as the content_store_document_type to search, and pub api is using esi_fund? Or have I got that mixed up?

Reply:
That is correct. I would suggest we should investigate if we can remove this in the future, but have chosen to leave it as it is for the time being